docs/pages/components/button.md

```diff
@@ -173,28 +177,135 @@ Button with multiple actions
+<button class="fd-button fd-button--standard fd-button--compact">Edit</button>
+<button class="fd-button fd-button--ghost fd-button--compact">Edit</button>
+<button class="fd-button fd-button--emphasized fd-button--compact sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--transparent fd-button--compact sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--positive fd-button--compact sap-icon--accept">Approve</button>
+<button class="fd-button fd-button--negative fd-button--compact sap-icon--decline">Reject</button>
+<button class="fd-button fd-button--attention fd-button--compact sap-icon--warning">Attention</button>
+{% endcapture %}
+{% include display-component.html component=button %}
+
+<br>
+
+## Icon and Text
+
+The buttons can have icon and/or text. The recommendation is to have either text or icon. Use icon for buttons only for very basic standard icon metaphores (e.g. _Back to previous screen, Create a new item, Remove from list, Edit, ..._)
+
+All buttons styles can be used with an icon. You can use the `sap-icon--{icon-name}` class to attach an icon to the button.
+Full list of all the available icons can be found on the <a href="icon.html">icons page</a>.
+{% capture button %}
+<button class="fd-button fd-button--emphasized sap-icon--cart">Add to Cart</button>
+<button class="fd-button sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--transparent sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--standard sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--ghost sap-icon--cart">Add to Cart</button>
+<button class="fd-button fd-button--positive sap-icon--accept">Approve</button>
+<button class="fd-button fd-button--negative sap-icon--decline">Reject</button>
+<button class="fd-button fd-button--attention sap-icon--warning">Attention</button>
+<br><br>
+<button class="fd-button sap-icon--cart"></button>
+<button class="fd-button fd-button--transparent sap-icon--cart"></button>
+<button class="fd-button fd-button--standard sap-icon--filter"></button>
+<button class="fd-button fd-button--ghost sap-icon--filter"></button>
+<button class="fd-button fd-button--positive sap-icon--accept"></button>
+<button class="fd-button fd-button--negative sap-icon--decline"></button>
+<button class="fd-button fd-button--attention sap-icon--decline"></button>
+{% endcapture %}
+{% include display-component.html component=button %}
+
+## Standard button state
+
+The buttons can also be set to a state:
+
+- **Normal**: The default state of the button. It can be clicked/tapped to perform the corresponding action.
+- **Selected**: Used to signal this button is selected among other buttons. This state can be rendered using `is-selected` class or `aria-selected="true"` attribute for accessibility.
+- **Disabled**: It cannot be clicked/tapped. This state can be rendered using `is-disabled` class or `aria-disabled="true"` attribute for accessibility.
```

Comment:
**Disabled**: It cannot be clicked/tapped. This state can be rendered using the `is-disabled` class or the `aria-disabled="true"` attribute for accessibility.